servers/cromwell/jobs/controllers/jobs_controller.py

```diff
@@ -607,15 +600,15 @@ def tail_file_contents(bucket, object, **kwargs):
     headers = {
         'Content-Type': 'text/xml',
         'Authorization': 'Bearer ' + token,
-        'Range': 'bytes=-20000'
+        'Range': 'bytes=-500000'
```

Comment:
That's quite a bump... I assume we aren't too concerned about security or money since this is a manual and opt-in thing to be doing?